tool/debian/singa-cuda8.0-cudnn5/DEBIAN/control

```diff
@@ -0,0 +1,14 @@
+Package: singa-cuda8
+Version: 1.0.1
```

Comment:
The version ID is set correctly in CMakeLists.txt: SET(PACKAGE_VERSION "1.1.0"). No need to update any thing. The Debian package will be built with the correct version from CMakeLists.txt.